### PR TITLE
Refactor AWSSQSSource adapter to use eventing lib

### LINKED ISSUE #81: Refactor awssqs source to use Knative's adapter package
Should include necessary changes to Makefile, CI conf, and docs.

### PR DESCRIPTION
**Pull request by antoineco**
Closes #81

### CODE REVIEW

File: pkg/adapter/awssqssource/adapter_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2019 TriggerMesh, Inc
+Copyright (c) 2020 TriggerMesh Inc.
```
Comment (sameersbn) [inline pkg/adapter/awssqssource/adapter_test.go:2]: nitpick: 
```suggestion
Copyright (c) 2019-2020 TriggerMesh Inc.
```

Comment (antoineco) [reply pkg/adapter/awssqssource/adapter_test.go:2]: Will create a follow up task for it because we'll need to update them all.

Comment (sameersbn) [reply pkg/adapter/awssqssource/adapter_test.go:2]: fair enough!

Comment (antoineco) [reply pkg/adapter/awssqssource/adapter_test.go:2]: Done! #103

**Pull request merged**